frontend/libs/studio-components/src/components/StudioContentMenu/StudioContentMenuLinkTab/StudioContentMenuLinkTab.tsx

```diff
@@ -18,5 +18,9 @@ export function StudioContentMenuLinkTab<TabId extends string>({
 }: StudioContentMenuLinkTabProps<TabId>): React.ReactElement {
   const props = useTabProps(icon, tabName, tabId);
 
-  return <StudioButton asChild>{renderTab(props)}</StudioButton>;
+  return (
+    <Button size='sm' asChild>
+      {renderTab(props)}
+    </Button>
+  );
```

Comment:
Why are we not using `StudioButton` with `as` prop here?

Reply:
Because the component is based on using the `asChild` functionality. We need to refactor all the usages of this component so that it will receive a `href` prop, and create a wrapper for it similar to `ButtonRouterLink` so that we can implement a `to` prop. That will be a separate piull request.